### ISSUE TITLE
login with OTT facing issue

### ISSUE THREAD

**Issue by keerthankumar** [kalilinux-png/Desktop#1]
{'fault': {'code': 999024, 'description': 'https://ctradeapi.kotaksecurities.com/', 'message': 'Request can not be processed, please process your request through provided domain'}}

**Comment by kalilinux-png** [comment]
Hi, @keerthankumar 

I tried some test cases and tried to figure out the issue but all test cases were passed successfully
It would be great if you could add some snapshot of your code so that our team can have a deep view of this issue 

Thanks 
Shubham Yadav

**Comment by keerthankumar** [comment]
i dint make any changes i run with exact same code. (OTT login)

**Comment by keerthankumar** [comment]
it says - 'message': 'Request can not be processed, please process your request through provided domain' . Does kotak provide any domain to use ?

**Comment by kalilinux-png** [comment]
@keerthankumar No Kotak doesn't provide any domain I think we need to evaluate the app with Kotak's API manager website

**Comment by keerthankumar** [comment]
looks like we cant create a new application. let me contact kotak

**Comment by kalilinux-png** [comment]
@keerthankumar Yes Great Idea and let me know what they say because this is the same issue one of my client was also facing

**Comment by keerthankumar** [comment]
I just did, looks like they are having some issue. Users cant create new applications for now.

**Comment by kalilinux-png** [comment]
@keerthankumar  yes this is not something new they are not allowing to create new apps for anyone 
I will also try to figure out why you are facing this issue and will try to resolve it

**Comment by keerthankumar** [comment]
Btw were you able to fetch the user account fund?

**Comment by kalilinux-png** [comment]
> Btw were you able to fetch the user account fund?

Yes @kalilinux-png I am able to fetch the user account funds and for you, I have added the code to the login with the ott file you just need to do some JSON slicing

**Comment by keerthankumar** [comment]
looks like the repo I was referring to was an old one.
https://github.com/osparamatrix/ks-orderapi-python/issues/57#issuecomment-1127664642

Here you don't need additional logic for the OTT.
